content/src/storage/ContentStorage.ts

```diff
@@ -1,24 +1,34 @@
-import { ContentFileHash } from 'dcl-catalyst-commons'
 import { Readable } from 'stream'
+import { createGunzip } from 'zlib'
 
 export type ContentEncoding = 'gzip'
 
 export interface ContentStorage {
-  storeStream(id: string, content: Readable): Promise<void>
-  /** @deprecated use storeStream instead */
-  store(id: string, content: Uint8Array): Promise<void>
-  delete(ids: string[]): Promise<void>
-  retrieve(id: string): Promise<ContentItem | undefined>
-  exist(ids: string[]): Promise<Map<string, boolean>>
-  stats(id: string): Promise<{ size: number } | undefined>
-  storeContent(fileHash: ContentFileHash, content: Uint8Array | Readable): Promise<void>
-  size(fileHash: ContentFileHash): Promise<number | undefined>
+  storeStream(fileId: string, content: Readable): Promise<void>
+  delete(fileIds: string[]): Promise<void>
+  retrieve(fileId: string): Promise<ContentItem | undefined>
+  exist(fileId: string): Promise<boolean>
+  existMultiple(fileIds: string[]): Promise<Map<string, boolean>>
+}
+
+export type RawContentItem = {
```

Comment:
nit: could it be possible to name it consistently with the `asRawStream`? Like `ContentRawStream`, `RawStream` or something alike?